src/main/java/com/frameworkium/core/ui/driver/AbstractDriver.java

```diff
@@ -31,7 +31,11 @@
      */
     @Override
     public void tearDown() {
-        this.webDriverWrapper.quit();
+        if (Property.REUSE_BROWSER.isSpecified()) {
```

Comment:
Are we assuming only browsers (i.e. not Appium etc.) here, if so maybe we should have some validation that the `-Dbrowser` is set to something which does indeed behave like a browser? Not sure if this is actually a problem but worth trying out if you (or anyone) has any Appium tests or something which isn't a web browser?